Server/Server.Test/ArgParserTests.cpp

```diff
@@ -151,3 +155,27 @@ TEST_CASE("Method of obtaining a pair of arguments of the argparser [ArgParser][
         CHECK_THROWS_AS(parser.getPair("any_flag"), std::exception);
     }
 }
+
+TEST_CASE("Method for generating an object of the Settings class [ArgParser][Success]")
```

Comment:
Please stop using tags worng!
https://github.com/catchorg/Catch2/blob/devel/docs/test-cases-and-sections.md#tags

Reply:
Didn't we say that these tags are valid? If that has changed, let everyone know.